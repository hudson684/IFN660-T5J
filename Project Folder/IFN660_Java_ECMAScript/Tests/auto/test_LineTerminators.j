asaadaded 
Enter

	whitespace text \f text
