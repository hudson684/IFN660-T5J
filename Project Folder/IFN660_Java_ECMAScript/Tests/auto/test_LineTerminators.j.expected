IDENT (asaadaded)
IDENT (Enter)
IDENT (whitespace)
IDENT (text)
EscapeSequence (\f)
IDENT (text)
EOF
